q.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
What are these for?

Reply:
These actually weren't supposed to be merged, this was for my use.